src/Process/Deploy/PreDeploy/CheckEnvFile.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Process\Deploy\PreDeploy;
+
+use Magento\MagentoCloud\App\GenericException;
+use Magento\MagentoCloud\Config\Deploy\Reader;
+use Magento\MagentoCloud\Config\Deploy\Writer;
+use Magento\MagentoCloud\Config\State;
+use Magento\MagentoCloud\Config\Validator\Deploy\DatabaseConfiguration;
```

Comment:
Let's not depend on validators in processes and vice versa, because those are different domain models. Please use separate class for database configuration validation

Reply:
What do you mean *separate class* for database configuration validation? 
Our validators were created for using in Processes and we already depending on them in processes.